src/ReplaySubject.ts

```diff
@@ -2,55 +2,50 @@ import {Subject} from './Subject';
 import {Scheduler} from './Scheduler';
 import {queue} from './scheduler/queue';
 import {Subscriber} from './Subscriber';
-import {TeardownLogic} from './Subscription';
+import {Subscription} from './Subscription';
 import {ObserveOnSubscriber} from './operator/observeOn';
 
 /**
  * @class ReplaySubject<T>
  */
 export class ReplaySubject<T> extends Subject<T> {
   private events: ReplayEvent<T>[] = [];
-  private scheduler: Scheduler;
   private bufferSize: number;
   private _windowTime: number;
 
   constructor(bufferSize: number = Number.POSITIVE_INFINITY,
               windowTime: number = Number.POSITIVE_INFINITY,
-              scheduler?: Scheduler) {
+              private scheduler: Scheduler = queue) {
     super();
-    this.scheduler = scheduler;
     this.bufferSize = bufferSize < 1 ? 1 : bufferSize;
     this._windowTime = windowTime < 1 ? 1 : windowTime;
```

Comment:
I feel like these should have a consistent naming scheme. Either use the prefixed '_' or don't.